src/components/simpleGrid/index.stories.tsx

```diff
@@ -23,54 +24,44 @@ const meta: Meta<typeof SimpleGridComponent> = {
 
   args: {
     columns: 3,
-    spacing: 'md',
+    gap: 'md',
     children: [
-      <Box h="50px" bg="blue.100" key="1">
+      <Box h="lg" bg="blue.100" key="1">
         1
       </Box>,
-      <Box h="50px" bg="green.100" key="2">
+      <Box h="lg" bg="green.100" key="2">
         2
       </Box>,
-      <Box h="50px" bg="orange.100" key="3">
+      <Box h="lg" bg="orange.100" key="3">
         3
       </Box>,
-      <Box h="50px" bg="blue.200" key="4">
+      <Box h="lg" bg="blue.200" key="4">
         4
       </Box>,
-      <Box h="50px" bg="green.200" key="5">
+      <Box h="lg" bg="green.200" key="5">
         5
       </Box>,
-      <Box h="50px" bg="orange.300" key="6">
+      <Box h="lg" bg="orange.300" key="6">
         6
       </Box>,
     ],
   },
 
   argTypes: {
-    spacing: {
-      options: ['xs', 'sm', 'md', 'lg', 'xl', '2xl', '3xl', '4xl'],
-
-      control: {
-        type: 'select',
-      },
-    },
-
-    spacingY: {
+    gap: {
       options: ['xs', 'sm', 'md', 'lg', 'xl', '2xl', '3xl', '4xl'],
 
       control: {
         type: 'select',
       },
     },
-
-    spacingX: {
+    rowGap: {
```

Comment:
Can we make this a token control? This should accept spacing tokens similar to margin and padding.
I'd do it globally here: https://github.com/smg-automotive/components-pkg/pull/1185/files#diff-c66cc7fc3967560e88f39979a0c3e683cac6a4bf73fe82f2ee4df63f1fd224eeR9